### PR TITLE
repo-sync-2024-03-12T19:05:39+0800

### PR DESCRIPTION
**Pull request by zhouaihui**
### Added

- SL: grad_average is supported on GPU.
- SL: mix_up defence can be called through callback.
- Simulation: add some unpartitioned datasets.

### Fixed

- Fix order in componet load table.

### Changed

- Serving: Update serving linkfunc type.
- FL: rename get/set_gradients methods
- SGX: improve doc for the accuracy analysis.
- SL: import fia attack.

### CODE REVIEW

File: examples/app/h_cv/fedper/fedper.ipynb [hunk 1]
```diff
@@ -1,1087 +0,0 @@
-{
```
Comment (ian-huu) [inline examples/app/h_cv/fedper/fedper.ipynb:1]: 这个为什么会删掉

Comment (zhouaihui) [reply examples/app/h_cv/fedper/fedper.ipynb]: 已Revert

**Review by 6fj — APPROVED** [review]
lgtm

**Review by da-niao-dan — APPROVED** [review]
lgtm

**Review by ian-huu — APPROVED** [review]
LGTM

**Pull request merged**